### PR TITLE
Add Shipping Carrier to Facility

### PR DESCRIPTION
**Pull request by Amir-HC**
Added details about adding carrier to a facility

### CODE REVIEW

File: deployment&configuration/oms/user-and-gateway-config/ShippingGateways.md [hunk 1]
```diff
@@ -85,3 +85,16 @@ By performing these steps, you essentially map the Packaging box from HotWax Com
 {% embed url="https://youtu.be/9XefekAuxr8" %}
 Video: Remove Shipment Box
 {% endembed %}
+
+### Add Career Facility
+
+In HotWax Commerce, the ability to add shipping carriers to facilities is essential for order fulfillment. Once a shipment gateway is set, adding carriers to facilities enables rate shopping and shipping label generation specifically for those carriers associated with the selected facility. This feature enhances workflow efficiency by allowing the HotWax Commerce Store Fulfillment App to intelligently select the most cost-effective shipping carrier for each order. To add careers to a facility follow these steps:
+
+1. Navigate to the `Career Facility` section.
```
Comment (dt2patel) [inline deployment&configuration/oms/user-and-gateway-config/ShippingGateways.md]: I don't think this is correct @Amir-HC

**Review by dt2patel — CHANGES_REQUESTED** [review]
Check section naming

File: deployment&configuration/oms/user-and-gateway-config/ShippingGateways.md (hunk 1 above)
Comment (Amir-HC) [reply deployment&configuration/oms/user-and-gateway-config/ShippingGateways.md:93]: Screenshot 2024-02-06 at 6 56 50 PM

I checked this on demo instance

Comment (dt2patel) [reply deployment&configuration/oms/user-and-gateway-config/ShippingGateways.md]: Check spelling of carrier

**Review by dt2patel — APPROVED** [review]

**Pull request merged**